resources/public/api/conf/2.0/examples/uk_property_historic_fhl_period_summary_create_and_amend/consolidated_request.json

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
can the file just be auto-formatted in Intellij to remove the empty top line?
(Also technically ought to have a newline at the end of the last line)

Reply:
Formatted